Lib/test/test_capi/test_opt.py

```diff
@@ -69,7 +69,8 @@ def loop():
                 self.assertEqual(opt.get_count(), 0)
                 with clear_executors(loop):
                     loop()
-                self.assertEqual(opt.get_count(), 1000)
+                # Subtract 16 because optimizer doesn't kick in until 16
+                self.assertEqual(opt.get_count(), 1000 - 16)
```

Comment:
```suggestion
                self.assertLessEqual(opt.get_count(), 1000 - _testinternalcapi.TIER2_THRESHOLD)
```
We are bound to change the threshold, so it would be good not to hard code it in the tests.